### PR TITLE
{chem}[intel/2021b] FHI-aims v210716_3

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/FHI-aims/FHI-aims-210716_3-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+maintaining high numerical accuracy for all production tasks. Production
+calculations handle up to several thousand atoms and can efficiently use (ten)
+thousands of cores. 
+"""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
+toolchainopts = {'opt': True, 'precise': True}
+
+# The source code must be downloaded manually from the FHI-aims club (https://aimsclub.fhi-berlin.mpg.de/).
+# Access to the FHI-aims club requires a valid license and registration. Details on available license options
```
Comment (jfgrimm) [inline easybuild/easyconfigs/f/FHI-aims/FHI-aims-210716_3-intel-2021b.eb:28]: it would be great to make use of `download_instructions` for this

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by sassy-crick** [comment]
Test report by @sassy-crick
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
hp - Linux Debian GNU/Linux 11, x86_64, 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz, Python 3.9.2
See https://gist.github.com/89c7c429cef291156a0f31dc53963270 for a full test report.

**Comment by sassy-crick** [comment]
Test report by @sassy-crick
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
hp - Linux Debian GNU/Linux 11, x86_64, 11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz, Python 3.9.2
See https://gist.github.com/7c0a2ab2174b8e69c1a694b33718804d for a full test report.

**Comment by sassy-crick** [comment]
As there are newer versions of the program out, I will close this PR

**Pull request closed without merging**